app/models/world_location_news_translation.rb

```diff
@@ -0,0 +1,11 @@
+class WorldLocationNewsTranslation
+  attr_accessor :translated_locales
+
+  def initialize(translated_locales = [])
+    @translated_locales = [].push(translated_locales).flatten
```

Comment:
I think it should always be an array, but I thought some defensive coding might be better than assumptions - fixed in 3b7e179